incuna_auth/middleware/permission_feincms.py

```diff
@@ -35,7 +35,10 @@ def _get_page_from_path(self, path):
         Override this to deal with different types of object from Page.
         """
         from feincms.module.page.models import Page
-        return Page.objects.best_match_for_path(path)
+        try:
+            return Page.objects.best_match_for_path(path)
+        except Page.DoesNotExist:
+            return None
```

Comment:
You need the `from feincms.module.page.models import Page`, which is why this method exists in the first place.  Mocking this method out in the tests allows us to avoid installing the entirety of FeinCMS to do some tests.

At some point in the future this section of code will be moved to `feincms-extensions`, so for now the slightly kludgy version is OK.